pupa/cli/commands/update.py

```diff
@@ -59,6 +59,46 @@ def save_report(report, jurisdiction):
             )
 
 
+@transaction.atomic
+def save_report(report, jurisdiction):
+    from pupa.models import RunPlan
+    plan = RunPlan.objects.filter(jurisdiction=jurisdiction).latest('id')
```

Comment:
Any reason we need to decouple like that?

I guess it makes sense if it's run without import -- why not set the run_plan to None, and if it's None, do a DB lookup, otherwise use the object from above?